integration-test/2022-disputed-borders.py

```diff
@@ -126,3 +130,154 @@ def test_admin_level_3_other_place(self):
             z, x, y, 'boundaries', {
                 'id': 345678,
             })
+
+    def test_disputed_by_to_unrecognized_disputed_reference_line(self):
+        z, x, y = (16, 53533, 28559)
+
+        self.generate_fixtures(
+            # https://www.openstreetmap.org/relation/13574166
+            dsl.way(13574166, dsl.tile_diagonal(z, x, y), {
+                'admin_level': '2',
+                'boundary': 'disputed',
+                'disputed_by': 'RU;PK;IL;PS;SA;EG;ID;BD',
+                'name': '1949 Israeli-Syrian DMZ',
+                'ne:brk_a3': 'B16',
+                'ne_id': '1746705859;1746705871',
+                'type': 'linestring',
+            }),
+        )
+
+        self.assert_has_feature(
+            z, x, y, 'boundaries', {
+                'id': 13574166,
+                'disputed': True,
+                'kind:ru': 'unrecognized_disputed_reference_line',
+                'kind:pk': 'unrecognized_disputed_reference_line',
+                'kind:il': 'unrecognized_disputed_reference_line',
+                'kind:ps': 'unrecognized_disputed_reference_line',
+                'kind:sa': 'unrecognized_disputed_reference_line',
+                'kind:eg': 'unrecognized_disputed_reference_line',
+                'kind:id': 'unrecognized_disputed_reference_line',
+                'kind:bd': 'unrecognized_disputed_reference_line',
+                'kind': 'disputed_reference_line',
+                'kind_detail': '2',
+            })
+
+    def test_boundary_claim_disputed_by_claimed_by(self):
+        z, x, y = (16, 53533, 28559)
+
+        self.generate_fixtures(
+            # https://www.openstreetmap.org/relation/13574166
+            dsl.way(202058477, dsl.tile_diagonal(z, x, y), {
+                'admin_level': '2',
+                'boundary': 'claim',
+                'claimed_by': 'CN;TW',
+                'disputed_by': 'IN;PK;TR',
+                'name': 'Extent of Chinese Claim - ﻿Bara Hoti area',
+                'name:de': 'Ausdehnung des chinesischen Anspruchs',
+                'name:en': 'Extent of Chinese Claim - ﻿Bara Hoti area',
+                'name:zh': '中国声称边境线',
+                'name:zh_pinyin': 'Zhōngguó shēngchēng biānjìng xiàn',
+                'ne:brk_a3': 'B02',
+                'ne_id': '1746705405',
+                'recognized_by': 'RU',
+                'type': 'linestring',
+            }),
+        )
+
+        self.assert_has_feature(
+            z, x, y, 'boundaries', {
+                'id': 202058477,
+                'kind': 'unrecognized_country',
```

Comment:
It would be easier to reason about this in a stylesheet with `kind: disputed_claim` (see ETL comments below).